### PR TITLE
PAE-425.1 - Change the Proversity references to Pearson.

### PR DESCRIPTION
**Pull request by Squirrel18**
## Description:

This PR changes all references from Proversity to Pearson as we have moved to a brand new GitHub organization.

## Notes:

Report backend settings and API URLs will be affected after this PR is merged.

**API changes:** All report API URLs changed from /proversity-reports/* to /pearson-reports/*.

**Python Module:** Main module was changed from openedx_proversity_reports* to openedx_pearson_reports*.

Authoring and contact information has been changed from Proversity to eduNEXT.

## Reviewers:

- [ ] @diegomillan 
- [ ] @luismorenolopera

### CODE REVIEW

File: openedx_pearson_reports/api/v0/views.py [hunk 1]
```diff
@@ -14,23 +14,23 @@
 from rest_framework.reverse import reverse
 from rest_framework.views import APIView
 
-from openedx_proversity_reports.edxapp_wrapper.openedx_authentication import \
+from openedx_pearson_reports.edxapp_wrapper.openedx_authentication import \
```
Comment (diegomillan) [inline openedx_pearson_reports/api/v0/views.py:17]: A minor suggestion to consolidate code styling, either use:
```suggestion
from openedx_pearson_reports.edxapp_wrapper.openedx_authentication import (
    openedx_bearer_authentication
)
```
or
```suggestion
from openedx_pearson_reports.edxapp_wrapper.openedx_authentication import openedx_bearer_authentication
```

File: openedx_pearson_reports/api/v0/views.py [hunk 2]
```diff
@@ -14,23 +14,23 @@
...
     openedx_bearer_authentication
-from openedx_proversity_reports.edxapp_wrapper.get_edx_rest_framework_extensions import \
+from openedx_pearson_reports.edxapp_wrapper.get_edx_rest_framework_extensions import \
```
Comment (diegomillan) [inline openedx_pearson_reports/api/v0/views.py:19]: Same as above

File: openedx_pearson_reports/api/v0/views.py [hunk 3]
```diff
@@ -14,23 +14,23 @@
...
     get_jwt_authentication
-from openedx_proversity_reports.edxapp_wrapper.get_student_account_library import \
+from openedx_pearson_reports.edxapp_wrapper.get_student_account_library import \
```
Comment (diegomillan) [inline openedx_pearson_reports/api/v0/views.py:21]: Same as above

**Review by diegomillan — CHANGES_REQUESTED** [review]

File: openedx_pearson_reports/api/v1/views.py [hunk 4]
```diff
@@ -12,12 +12,12 @@
 from rest_framework.reverse import reverse
 from rest_framework.views import APIView
 
-from openedx_proversity_reports.edxapp_wrapper.openedx_authentication import \
+from openedx_pearson_reports.edxapp_wrapper.openedx_authentication import \
```
Comment (diegomillan) [inline openedx_pearson_reports/api/v1/views.py:15]: Same suggestion as v0 version @Squirrel18

File: openedx_pearson_reports/apps.py [hunk 5]
```diff
@@ -1,15 +1,15 @@
...
+File configuration for openedx-pearson-reports.
 """
 from django.apps import AppConfig
 
 
-class OpenEdxProversityReportsConfig(AppConfig):
+class OpenEdxPearsonReportsConfig(AppConfig):
     """
-    Plugin app configuration for openedx-proversity-reports
+    Plugin app configuration for openedx-pearson-reports
```
Comment (luismorenolopera) [inline openedx_pearson_reports/apps.py:9]: @Squirrel18 add trailing comma.

File: openedx_pearson_reports/api/v0/views.py (hunk 1 above)
Comment (Squirrel18) [reply openedx_pearson_reports/api/v0/views.py]: Not the purpose of this RP, but applied... @diegomillan

File: openedx_pearson_reports/api/v0/views.py (hunk 2 above)
Comment (Squirrel18) [reply openedx_pearson_reports/api/v0/views.py]: Not the purpose of this PR, but applied... @diegomillan

File: openedx_pearson_reports/api/v0/views.py (hunk 3 above)
Comment (Squirrel18) [reply openedx_pearson_reports/api/v0/views.py]: Not the purpose of this PR, but applied... @diegomillan

File: openedx_pearson_reports/api/v1/views.py (hunk 4 above)
Comment (Squirrel18) [reply openedx_pearson_reports/api/v1/views.py]: Not the purpose of this PR, but applied... @diegomillan

File: openedx_pearson_reports/apps.py (hunk 5 above)
Comment (Squirrel18) [reply openedx_pearson_reports/apps.py:9]: What? @luismorenolopera

**Review by diegomillan — APPROVED** [review]

**Review by luismorenolopera — APPROVED** [review]
LGTM

**Pull request merged**